tests/forte/data/data_store_test.py

```diff
@@ -231,31 +263,64 @@ def test_get(self):
         self.assertEqual(instances[1][2], 3456)
 
         # get all entries
+<<<<<<< HEAD
+        instances = list(self.data_store.get(2))
+        self.assertEqual(len(instances), 5)
+=======
         instances = list(self.data_store.get("forte.data.ontology.core.Entry"))
         self.assertEqual(len(instances), 4)
+>>>>>>> master
```

Comment:
please be careful and never push git diff syntax to code, it can break a lot of thing